src/pattern-library/components/patterns/UtilitiesPage.js

```diff
@@ -53,26 +53,7 @@ export default function UtilitiesPage() {
           This package also provides some tailwind variants (modifiers) that can
           be combined with other tailwind utility classes.
         </p>
-        <Library.Example title="ARIA variants">
```

Comment:
Note: These examples would behave correctly still as written, but there is no need to document them as a special utility as they are core to Tailwind now.